AEPSignal/Sources/SignalHitProcessor.swift

```diff
@@ -70,6 +70,10 @@ class SignalHitProcessor: HitProcessing {
             // retry this hit later
             Log.debug(label: LOG_TAG, "Signal request failed with recoverable error \(connection.error?.localizedDescription ?? "") and status code \(connection.responseCode ?? -1). Will retry sending the request later: \(hit.url.absoluteString)")
             completion(false)
+        } else if let error = connection.error as? URLError, error.isRecoverable {
+            // retry this hit later as the request failed with a recoverable transport error
+            Log.debug(label: LOG_TAG, "Signal request failed with unrecoverable error \(connection.error?.localizedDescription ?? "") and status code \(connection.responseCode ?? -1). It will be dropped from the queue: \(urlString)")
```

Comment:
this log is inaccurate, the hit is retried in the case of a recoverable error code